dpos/manager/dposmanager.go

```diff
@@ -114,6 +120,8 @@ type DPOSManager struct {
 	recoverStarted       bool
 	notHandledProposal   map[string]struct{}
 	statusMap            map[uint32]map[string]*dmsg.ConsensusStatus
+
```

Comment:
Remove blank line.

Reply:
"neededMajorityStatus" "recoverStarted" "notHandledProposal" "statusMap" is for recover abnormal state,  requestedBlocks if for broadcast block for dpos network.  “requestedBlocks” should be separated.